skins/oasis/css/core/ads.scss

```diff
@@ -281,6 +281,11 @@ body >a >img[src$="noad.gif"] {
   }
 }
 
+.ad-in-content {
```

Comment:
@gabrys Should it be just margin-left and margin-bottom? I can see there's a logic in Venus that sometimes adds a margin-right but it might be skin-specific.